src/source/promises.bs

```diff
@@ -470,6 +470,20 @@ namespace promises
         return promises.resolve(value)
     end function
 
+    ' Sets a global flag to enable or disable logging of crashes when calling callback functions
+    function setCrashLogging(enabled as boolean) as boolean
```

Comment:
What about calling this `enableCrashLogging` instead? 
```suggestion
    function enableCrashLogging(enabled as boolean) as boolean
```

I also wonder if it would be nicer moving this to some sub namespace. like `promises.configuration.enableCrashLogging` so it's out of the way of the primary functions?